circleguard/settings.py

```diff
@@ -227,6 +227,7 @@
         "cache_dir": QStandardPaths.writableLocation(QStandardPaths.AppDataLocation) + "/cache/",
         "log_dir": QStandardPaths.writableLocation(QStandardPaths.AppDataLocation) + "/logs/",
         "config_location": QStandardPaths.writableLocation(QStandardPaths.AppDataLocation),
+        "appdata_location": QStandardPaths.writableLocation(QStandardPaths.AppDataLocation),
```

Comment:
I meant this to be a python-level variable at the top of the file, not a new settings entry. Then all these other settings (cache_dir/etc) can also use it:

```python
appdata_dir = QStandardPaths.writableLocation(QStandardPaths.AppDataLocation)

...

"cache_dir": appdata_dir + "/cache/",
# etc
```

and it would be imported in gui.py as a normal python import instead of accessing a setting. No reason to add a new setting when we don't need to